### PR TITLE
Feature/1511 1414 fix user profile bugs

### PR DESCRIPTION
**Pull request by fcv-iteratorIt**

### CODE REVIEW

File: src/services/user-management/user.service.ts [hunk 1]
```diff
@@ -90,17 +90,13 @@ export class UserService {
         });
     }
 
-    async findOne(
-        id: number,
-        getPermissionOrganisationInfo = false,
-        getPermissionUsersInfo = false
-    ): Promise<User> {
+    async findOne(id: number, getExtendedInformation: boolean = false): Promise<User> {
```
Comment (augusthjerrild) [inline src/services/user-management/user.service.ts:93]: The type annotation "boolean" should be removed, since "false" is making it boolean

**Review by augusthjerrild — APPROVED** [review]

File: src/services/user-management/user.service.ts (hunk 1 above)
Comment (fcv-iteratorIt) [reply src/services/user-management/user.service.ts:93]: While true, I still prefer the strong typing of having it explicitly set

Comment (augusthjerrild) [reply src/services/user-management/user.service.ts:93]: Fair - in vs code (at least in mine) it will show as an error though :-)

**Comment by ramogens** [comment]
Fixes https://github.com/OS2iot/OS2IoT-frontend/issues/119

**Pull request merged**